frontend/src/ui/components/atoms/event-card/EventCard.spec.tsx

```diff
@@ -0,0 +1,41 @@
+import EventCard from '@/ui/components/atoms/event-card/EventCard';
+import { render, screen } from '@testing-library/react';
+
+const defaultProps = {
+  title: 'Pizza Day',
+  month: 'January',
+  day: '01',
+  weekday: 'Monday',
+  colour: '#000000',
+};
+
+describe('EventCard', () => {
+  it('should render all the texts passed by props', () => {
+    render(<EventCard {...defaultProps} />);
+    const RightButton = screen.queryByRole('button', { name: 'SVG right' });
+    const eventCardDate = screen.getByTestId('event-card-date');
```

Comment:
You dont need to use getByTestId. You have already getByText below (which is more appropiate

Reply:
Explained in the other comment, please tell me what do you think

Reply:
Perfect!, This is the reason that i want to listen, you really are checking the background color, for this component. Not the container as you said in the beginning :)

Reply:
thanks!!